index.html

```diff
@@ -1361,6 +1360,8 @@
 
 [^normalization]:  Omitting normalization factors.
 
+[^anisotropy_g]: Technically, $g$ is the _mean-cosine of deflection_ of the phase-function, which is not specific to the Henyey-Greenstein phase function model [#d'Eon2021].
```

Comment:
Just checking why you decided to put this in a footnote instead of mentioning the "mean cosine" in the main text. Is it because the anisotropy is discussed in two different sections (for subsurface and interior volumes)?

Reply:
> mean-cosine

> phase-function

These shouldn't be hyphenated. 

("mean" is simply an adjective describing "cosine". And "phase function" would only be hyphenated if being used as a compound adjective, however we decided to not hyphenate even compound adjectives if they are terms of art.)

Reply:
>Just checking why you decided to put this in a footnote instead of mentioning the "mean cosine" in the main text. Is it because the anisotropy is discussed in two different sections (for subsurface and interior volumes)?

Just because this is a technical detail that seems out of place in the main text. It's not a fact you even to know to implement the Henyey-Greenstein model.

Reply:
>These shouldn't be hyphenated.

Fixed in fc4c5d25705bb7560751b65b18801d44c1c4ef83.

Reply:
Makes sense.